src/CommandingOfficers/CommanderBear_Bull.java

```diff
@@ -17,9 +17,30 @@
 {
   private static final long serialVersionUID = -3048055535977630509L;
   
-  private static final CommanderInfo coInfo = new CommanderInfo("Bear&Bull", new instantiator());  
-  private static class instantiator implements COMaker
+  private static final CommanderInfo coInfo = new CommanderInfo("Baron Bull", new instantiator());  
```

Comment:
Aww, no more dual-portrait potential.

Reply:
I mean, I'm quite willing to swap it back.
....I think I might have switched it due to being freaked out about having the name as the save file name?

Reply:
Aww, no more clever pun.

Reply:
I really can't win here, can I?

Reply:
I did put it in a separate commit so it can be reverted easily, though

Reply:
Nope, no winning on my watch.

I really don't have a stake in the matter; he's your Commander, do whatchu want.